src/main/java/arb/logic/parser/CliSyntax.java

```diff
@@ -10,6 +10,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_PHONE = new Prefix("phone/");
     public static final Prefix PREFIX_EMAIL = new Prefix("email/");
 
-    //public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_TAG = new Prefix("t/");
```

Comment:
Just to confirm, you meant to do Prefix("t/") and not Prefix("tag/") right? Because I see that others have a word in their new Prefix() like Prefix("phone/") 😅